src/spaceone/dashboard/model/project_dashboard_model.py

```diff
@@ -21,7 +21,7 @@ def to_dict(self):
 
 class ProjectDashboard(MongoModel):
     project_dashboard_id = StringField(max_length=40, generate_id='project-dash', unique=True)
-    name = StringField(max_length=255)
+    name = StringField(max_length=255, unique_with='project_id')
```

Comment:
Need to create compound unique index.
- domain_id
- project_id
- name

Please change code as follows.
`unique_with=['project_id', 'domain_id']`